README.md

```diff
@@ -1,2 +1,34 @@
-# books-with-es6
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Books list with ES6
+
+> Updatable books list coded with ES6 features
+
+
+## Built With
+
+- HTML, CSS and Javascript
+
```

Comment:
- [ ] (OPTIONAL) It would be great if you add a section in your README file explaining how a user can set up and run your project locally on their machine. This will make your README file more professional 👍

Reply:
thanks!